websocket_server.go

```diff
@@ -346,15 +346,29 @@ func (n *pushServer) start(ctx context.Context) {
 					}
 
 					if n.cfg.pushServer.dispatchHandler != nil {
-						ok, err := n.cfg.pushServer.dispatchHandler.ShouldDispatch(session, event, eventSummary)
+						dispatch, err := n.cfg.pushServer.dispatchHandler.ShouldDispatch(session, event, eventSummary)
 						if err != nil {
+							//if 'ShouldDispatch' returns an error type that has the method 'ShouldCloseSocket() (bool, int)'
```

Comment:
style: add spaces between `//` and the text

Reply:
done